core/schema/http.graphqls

```diff
@@ -1,14 +1,4 @@
-
 extend type Query {
     "An http remote"
-    http(url: String!): HTTPRemote!
-}
-
-"An HTTP remote endpoint"
-type HTTPRemote {
-    "Download a tar-encoded directory from this remote"
-    directory: Directory!
```

Comment:
cc @shykes we simplified the API here to just return a `File` only since it wasn't clear how to turn an arbitrary URL into a full directory, let us know if we were misremembering the intention here

Reply:
Yes that's fine. For some reason I thought `llb.Http` supported unpacking a tar archive, and setting llb state to the resulting tree. But now I think I may have imagined that... Adding it to the "one day maybe" bucket. Thanks!

Reply:
That reminds me something as well.
I wonder if it wasn't some limitation we had with our CUE API?

Reply:
I believe Dockerfile's `ADD` statement has behavior like this, could be the source of the memory? It's implemented on a higher-level than LLB, so we can't use it for free here.